### PR TITLE
Fixed Renderer to handle properly Manifest and Blob responses.

### PR DESCRIPTION
**Pull request by ipanova**
closes #7620
https://pulp.plan.io/issues/7620

### CODE REVIEW

File: pulp_container/app/tasks/sync_stages.py [hunk 1]
```diff
@@ -330,7 +330,6 @@ def create_blob(self, man_dc, blob_data):
             url=blob_url,
             relative_path=digest,
             remote=self.remote,
-            extra_data={"headers": V2_ACCEPT_HEADERS},
```
Comment (ipanova) [inline pulp_container/app/tasks/sync_stages.py:333]: This line is not related to the fix, however sending manifest headers for the blob requests is wrong.

**Review by mdellweg — APPROVED** [review]

**Review by dkliban — APPROVED** [review]
